appcues/build.gradle

```diff
@@ -3,12 +3,16 @@ plugins {
     id 'kotlin-android'
     id 'kotlin-kapt'
     id 'org.jetbrains.dokka'
+    id 'jacoco'
+}
+
+jacoco {
+    toolVersion = "0.8.9"
 }
```

Comment:
I'm not sure if this part is required, it is also set in the spec repo and I think the plugin version defined in the project level build gradle defines it? Maybe harmless, but could explore removing also.